app/src/main/kotlin/com/simplemobiletools/flashlight/activities/BrightDisplayActivity.kt

```diff
@@ -27,40 +45,65 @@ class BrightDisplayActivity : SimpleActivity() {
                 WindowManager.LayoutParams.FLAG_FULLSCREEN
         )
 
-        useDynamicTheme = false
         super.onCreate(savedInstanceState)
-        setContentView(binding.root)
-        supportActionBar?.hide()
-        setBackgroundColor(config.brightDisplayColor)
-
-        binding.brightDisplayChangeColor.setOnClickListener {
-            ColorPickerDialog(this, config.brightDisplayColor, true, currentColorCallback = {
-                setBackgroundColor(it)
-            }) { wasPositivePressed, color ->
-                if (wasPositivePressed) {
-                    config.brightDisplayColor = color
-
-                    val contrastColor = color.getContrastColor()
-                    binding.brightDisplayChangeColor.apply {
-                        setTextColor(contrastColor)
-                        background.applyColorFilter(contrastColor)
-                    }
-                } else {
-                    setBackgroundColor(config.brightDisplayColor)
+        enableEdgeToEdgeSimple()
+        setContent {
+            AppThemeSurface {
+                val colorPickerDialogState = rememberAlertDialogState().apply {
+                    ColorPicker()
                 }
+
+                ScreenContent(colorPickerDialogState)
```

Comment:
Don't pass the state, rather pass a function that'll need to be invoked when you want to show it, it can be like 
```kotlin
ScreenContent(colorPickerDialogState::show)
```